internal/core/client.go

```diff
@@ -57,7 +57,9 @@ func createClient(buildInfo *BuildInfo, profileName string) (*scw.Client, error)
 	}
 
 	userAgent := "scaleway-cli/"
-	userAgent += buildInfo.Version.String()
+	if buildInfo.Version != nil {
```

Comment:
Should we have a `GetUserAgent` on the `BuildInfo`  ?

Reply:
Sure

Reply:
done